projects/plugins/protect/src/js/components/threats-list/paid-list.jsx

```diff
@@ -53,7 +53,7 @@ const ThreatAccordionItem = ( {
 			event.preventDefault();
 			setModal( {
 				type: 'FIX_THREAT',
-				props: { id, label, title, icon, severity, fixable },
+				props: { id, fixable, title, icon, severity },
```

Comment:
Using `title` makes sense to me, does this affect the `handleIgnoreThreatClick()` method above this one as well? i.e. can we potentially remove the `getLabel()` method entirely?

I'm not sure why the label was not displaying in the first place, since we're generating the value programmatically. Maybe an issue of not using callback/memo hooks? I'm just testing to verify that, and then I'll approve this PR :+1: :+1:

Reply:
I've gone through this in more detail now and discovered that we don't generate the `label` prop for the threats list in advance of rendering this particular modal, whereas we do in others. Getting rid of the `getLabel` methods doesn't appear to be a logical solution either as we use that property in a variety of other location in the scan UI. Therefor, I've opted to reassess and bring in the label prop to the `FixAllThreatsModal` by generating the property initially directly within the `useThreatsList` hook, rather than the location that list is passed to.